src/ocamlorg_frontend/pages/tutorial.eml

```diff
@@ -1,5 +1,9 @@
 open Ocamlorg
 
+let render_title (title: string) = 
+  let sanitized_title = Utils.slugify title in 
+  <h1 id="<%s sanitized_title %>" ><a class="anchor" aria-hidden="true" href="#<%s sanitized_title %>"></a><%s title %></h1>
```

Comment:
```suggestion
let render_title (tutorial: Data.Tutorial.t) = 
    <h1 id="<%s tutorial.slug %>" ><a class="anchor" aria-hidden="true" href="#<%s tutorial.slug %>"></a><%s tutorial.title %></h1>
```

Reply:
Sorry, I didn't realized this could be made simpler on first read

Reply:
No problem